packages/vscode-extension/src/webview/utilities/consts.ts

```diff
@@ -108,3 +113,14 @@ export const AndroidSupportedDevices: DeviceProperties[] = [
     maskImage: pixel6amask,
   },
 ] as const;
+
+export function mapIdToModel(deviceId: string): string {
+  const device = iOSSupportedDevices
```

Comment:
would be easier to just call `find` on one collection and then on the other. This will clone arrays and given it runs in a cell render function it may be executed a large number of times